src/main/webapp/script.js

```diff
@@ -12,19 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-google.charts.load('current', {'packages':['corechart']});
-google.charts.setOnLoadCallback(getChart);
-
 const CHART_WIDTH = 800;
 const CHART_HEIGHT = 400;
+const HIGHEST_SCORE = 1.0;
+const LOWEST_SCORE = -1.0;
+const INTERVAL = 0.2;
+
+google.charts.load('current', {'packages':['corechart']});
+google.setOnLoadCallback(getChart)
 
 function getYouTubeComments() { 
   const urlInput = document.getElementById('url-entry');
   const url = cleanseUrl(urlInput.value);
   fetch("/YouTubeComments?url="+url)
       .then(response => response.json()).then((comments) =>{
```

Comment:
I'm not sure that we need the second `then` in this case since `response.json()` already returns a `Promise`.

We should be able to just change this to 

```
fetch("/YouTubeComments?url="+url)
      .then(response => response.json())
```

Reply:
You could clean this up by using asnyc/await here. That would simplify your promises and keep your function name accurate.
```javascript
async function getYoutubeComments() {
    const urlInput = document.getElementById('url-entry');
    const response = await fetch("/YouTubeComments?url="+url);
    const comments = await response.json();
    return comments;
}
```